src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -635,14 +655,20 @@ Currently, in order to export a File PFB, \`enableLimitedFilePFBExport\` must be
     });
   };
 
-  isFileButton = (buttonConfig) => buttonConfig.type === 'manifest'
-    || buttonConfig.type === 'export'
-    || buttonConfig.type === 'export-to-seven-bridges'
-    || buttonConfig.type === 'export-to-workspace'
-    || buttonConfig.type === 'export-to-pfb'
-    || buttonConfig.type === 'export-pfb-to-workspace';
+  isFilePFBButton = (buttonConfig) => buttonConfig.type === 'export-file-pfb-to-url'
+    || buttonConfig.type === 'export-files-to-pfb'
+    || buttonConfig.type === 'export-files' // deprecated
+    || buttonConfig.type === 'export-files-to-seven-bridges'; // deprecated
```

Comment:
if they are deprecated, should we just remove related codes? or there is a plan for that

Reply:
I think it's best to leave these button types in at least for the moment, it would be a breaking config change for a few commons (BDC, Anvil, BRH)